libpod/container.go

```diff
@@ -718,6 +728,21 @@ func (c *Container) Export(path string) error {
 	return err
 }
 
+// AddArtifacts creates and writes to an artifact file for the container
+func (c *Container) AddArtifacts(name string, data []byte) error {
```

Comment:
Nit: can you rename these to AddArtifact/GetArtifact/RemoveArtifact (make them singular, not plural?)

Reply:
fixed